### DISCUSSION TITLE
duda -Create_trip

### DISCUSSION THREAD

**Discussion by JoacoVillegas** [harpomaxx/algoritmos2#33] [General]
Buenas, ¿alguno puede confirmarme con que se refiere al inciso b el create_trip?, pensaba que se puede referir a una lista por las esquinas que tuvo que pasar para llegar a su destino, ¿o es el camino mas corto en peso?. Gracias.

**Comment by AgusOlivares** [comment]
Yo entendi que es la secuencia de esquinas para poder llegar al destino. ya que el peso luego lo utilizas en el costo del viaje

**Comment by harpomaxx** [comment]
SIendo que nuestro mapa es un grafo, entonces podemos considerar al camino como los nodos por los cuales tenemos que pasar para llegar a nuestro destino.
